src/main/java/com/microsoft/graph/http/IStatefulResponseHandler.java

```diff
@@ -47,11 +49,13 @@
      * @param logger     the logger
      * @param <ResponseType> the native http client response type
      * @return           the result generated by this handler
-     * @throws Exception an exception occurs if the request was unable to complete for any reason
+     * @throws IOException an exception occurs if the request was unable to complete for any reason
+     * @throws GraphServiceException an exception is there is an interruption related to the GraphService
      */
     @Nullable
+    @SuppressFBWarnings
     <ResponseType> ResultType generateResult(@Nonnull final IHttpRequest request,
                               @Nonnull final ResponseType response,
                               @Nonnull final ISerializer serializer,
-                              @Nonnull final ILogger logger) throws Exception;
+                              @Nonnull final ILogger logger) throws IOException, GraphServiceException;
```

Comment:
this is technically a binary breaking change. Anybody with a reference to that who's not currently handling those exceptions will get a compile time error when upgrading. Please revert the exception type change. Suppression will be enough as I believe this is going away anyway on the next version